src/config/Color.mdx

```diff
@@ -0,0 +1,47 @@
+import { Meta, ColorPalette, ColorItem } from '@storybook/blocks';
+import theme from '@/config/theme';
+
+<Meta title='Color' />
+
+# Color
+
+The swatches below are programmatically pulled from our theme, so they reflect the current state of the code,
+not the desired state of the theme discipline. See Figma for desired changes.
```

Comment:
Maybe we could make this comment a little more long-lived. Like simply "The swatches below reflect the default theme of the application" or something.